wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java

```diff
@@ -74,13 +81,51 @@ static String inferLmiEngine(ModelInfo<?, ?> modelInfo) throws ModelException {
             engineName = "DeepSpeed";
         } else if (isDeepSpeedRecommended(modelType)) {
             engineName = "DeepSpeed";
+        } else if (features != null && features.contains("trtllm")) {
+            engineName = "MPI";
         } else {
             engineName = "Python";
         }
         logger.info("Detected engine: {}, modelType: {}", engineName, modelType);
         return engineName;
     }
 
+    static void convertIfNeed(ModelInfo<?, ?> info) throws IOException {
+        String rollingBatch = info.prop.getProperty("option.rolling_batch");
```

Comment:
can we rely on `SERVING_FEATURES` to convert since conversion is needed without rollingBatch as well?

Reply:
ignore this comment, I see that's handled below.

Reply:
currently trtllm only work for rolling batch